### PR TITLE
fozzie-components@1.23.0 - Remove demo scripts in favour of Storybook / Update README.md's

### PR DESCRIPTION
**Pull request by siggerzz**

### CODE REVIEW

File: packages/f-header/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-header",
   "description": "Fozzie Header – Globalised Header Component",
-  "version": "3.3.0",
+  "version": "3.5.0",
```
Comment (tommcclean) [inline packages/f-header/package.json:4]: What happened to 3.4.0 🚀

Comment (siggerzz) [reply packages/f-header/package.json:4]: https://github.com/justeat/fozzie-components/pull/244

**Review by tommcclean — COMMENTED** [review]
LGTM - Except that I don't think we should bump the versions as we aren't going to release; happy to be pursuaded otherwise though.

**Review by ashleynolan — CHANGES_REQUESTED** [review]
I actually think it's worth keeping the separate demo components – my main reason is that there's no other way of debugging them through Vue dev tools as this doesn't work with storybook (I think because the component gets iframed in).

Happy to hear if there's a big reason for getting rid of them, but I think there's a use-case for keeping them in addition to the main storybook way of viewing them.

**Comment by siggerzz** [comment]
Not needed

**Pull request closed without merging**